rustual-boy-core/src/v810.rs

```diff
@@ -548,6 +618,9 @@ impl V810 {
                 }),
                 OPCODE_BITS_LDSR => format_ii!(|imm5, reg2| {
                     let value = self.reg_gpr(reg2);
+
+                    trace_add!(TraceEntry::SysReg(imm5, 0)); // what value?
```

Comment:
Perhaps I'm not understanding here, but is the issue that you don't have the value of the system reg from before this instruction executed? It should be possible to split out the code that gets its value into a fn that's used both in STSR and here to get that value, but perhaps I've misunderstood the issue.